data/items/items.xml

```diff
@@ -13347,15 +13347,21 @@
 	<item id="7670" article="a" name="flower pot">
 		<attribute key="description" value="The plant in there sprouted, but only gardeners might recognise what it will develop into." />
 		<attribute key="weight" value="150" />
+		<attribute key="decayTo" value="7678"/>
```

Comment:
Fixed it in your last PR, but the code style is " /> (note the space)

Reply:
something always goes unnoticed in my eyes